vision/diffusion_mnist/diffusion_plot.jl

```diff
@@ -0,0 +1,118 @@
+include("diffusion_mnist.jl")
+
+using Images
+
+"""
+Helper function yielding the diffusion coefficient from a SDE.
+"""
+diffusion_coeff(t, sigma=convert(eltype(t), 25.0f0)) = sigma .^ t
+
+"""
+Helper function that produces images from a batch of images.
+"""
+function convert_to_image(x, y_size)
+    Gray.(
+        permutedims(
+            vcat(
+                reshape.(
+                    chunk(x |> cpu, y_size), 28, :
+                )...
+            ),
+            (2, 1)
+        )
+    )
```

Comment:
The expression here can be split up into 2-3 lines without creating a pyramid. Though I feel there must be an easier way to do this conversion too...

Reply:
collapsed the pyramid. I guess extra typing is avoided in Julia? I feel the pyramid helps readability